puppet/modules/jenkins_job_builder/files/centos.org/jobs/foreman-infra-update-jenkins-jobs.yml

```diff
@@ -0,0 +1,10 @@
+---
+- job:
+    name: 'foreman-infra-update-jenkins-jobs'
```

Comment:
I guess this should be slightly reversed, we've tended to put the action at the end and the specifc -> generic at the front